ci/kind/kind-setup.sh

```diff
@@ -244,25 +245,68 @@ function configure_extra_networks {
   done
 }
 
+function get_bridge_id {
+  local bridge_id=$(docker network inspect kind -f {{.ID}})
+  local return_value=$?
+  if [ $return_value -eq 0 ]; then
+        echo "$bridge_id"
+    else
+        echo "Unable to get kind bridge id, Docker command failed with exit code $return_value"
+        exit 1
+    fi
+}
+
+# add_kind_ipam_routes  add routes for non-ipam test-pods.
+function add_kind_ipam_routes {
+     kubectl get nodes -o wide --no-headers=true |  awk '{print $1}' | while read nodes; do
+      local pod_cidr=$(kubectl get nodes "${nodes}" -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+      local node_ip=$(kubectl get nodes "${nodes}" -o wide | awk 'FNR == 2 {print$6}')
```

Comment:
This could be achieved via one query:
```
kubectl get nodes -o jsonpath='{range .items[*]}{.spec.podCIDR}{" "}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'
```